morphia/src/main/java/org/mongodb/morphia/annotations/Indexed.java

```diff
@@ -76,10 +76,7 @@
     boolean unique() default false;
 
     /**
-     * Indicates the direction of the index (ascending, descending); default is ascending
-     *
-     * @deprecated use the {@link IndexOptions} found in {@link #options()}
+     * Indicates the type of the index (ascending, descending); default is ascending
      */
-    @Deprecated
```

Comment:
Acknowledged, was it released as deprecated then?  What did users move to?  Or was it a mid release change and change back?  Either way, I'm happy that you're happy with it! :)